utilities/test_suite/HIP/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.5)
+project(test_suite/HIP)
+list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../../../cmake)
+set(ROCM_PATH /opt/rocm CACHE PATH "default ROCm installation path")
+list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH})
+
+# Set message options
+if(NOT WIN32)
+    string(ASCII 27 Esc)
+    set(ColourReset "${Esc}[m")
+    set(Red         "${Esc}[31m")
+    set(Green       "${Esc}[32m")
+    set(Yellow      "${Esc}[33m")
+    set(Blue        "${Esc}[34m")
+    set(BoldBlue    "${Esc}[1;34m")
+    set(Magenta     "${Esc}[35m")
+    set(Cyan        "${Esc}[36m")
+    set(White       "${Esc}[37m")
+endif()
+
+find_package(hip QUIET)
+find_package(OpenCV QUIET)
+find_package(TurboJpeg QUIET)
+
+# BOOST
+set(BOOST_COMPONENTS filesystem system)
+find_package(Boost COMPONENTS ${BOOST_COMPONENTS} REQUIRED)
+include_directories(${Boost_INCLUDE_DIRS} ${TurboJpeg_INCLUDE_DIRS})
+set(LINK_LIBRARY_LIST ${LINK_LIBRARY_LIST} ${Boost_FILESYSTEM_LIBRARY} ${Boost_SYSTEM_LIBRARY} ${TurboJpeg_LIBRARIES_DIR})
```

Comment:
Is turbojpeg needed in boost here? Pls check. You can add TurboJpeg_LIBRARIES_DIR to LINK_LIBRARY_LIST inside the if condition if its found.

Reply:
Done